### PR TITLE
fix for creating a new push profile with the same manifest

### PR DESCRIPTION
**Pull request by mihaibuzgau**
name and manifest file path is now relative to buildDirectory

### CODE REVIEW

**Comment by carabasdaniel** [comment]
:+1:

File: src/CloudFoundry.VisualStudio/ProjectPush/PushEnvironment.cs [hunk 1]
```diff
@@ -19,7 +22,8 @@ public PushEnvironment()
             this.targetFilePath = VsUtils.GetTargetFile();
 
             var project = VsUtils.GetSelectedProject();
-            this.profileFilePath = Path.Combine(VsUtils.GetPublishProfilePath(), "push.cf.pubxml");
+            this.profileFilePath = Path.Combine(VsUtils.GetPublishProfilePath(),
+                string.Format(CultureInfo.InvariantCulture, "{0}.cf.pubxml", DefaultProfileName));
```
Comment (viovanov) [inline src/CloudFoundry.VisualStudio/ProjectPush/PushEnvironment.cs]: Isn't there a helper function for this used everywhere?

File: src/CloudFoundry.VisualStudio/ProjectPush/PublishProfile.cs [hunk 2]
```diff
@@ -376,9 +376,24 @@ public static PublishProfile Load(PushEnvironment pushEnvironment)
...
         {
+
+            var profileDir = System.IO.Directory.GetParent(this.environment.ProfileFilePath).FullName;
+
+            if (string.IsNullOrWhiteSpace(this.name))
+            {
+                this.name = PushEnvironment.DefaultProfileName;
+            }
+
+            this.path = System.IO.Path.Combine(profileDir, string.Format(CultureInfo.InvariantCulture, "{0}.cf.pubxml", this.name));
```
Comment (viovanov) [inline src/CloudFoundry.VisualStudio/ProjectPush/PublishProfile.cs]: Use helper function, or constant that defines extension (.cf.pubxml)

**Comment by viovanov** [comment]
:+1:

**Pull request merged**